content/projects/tdd/password-checker/part2/_index.md

```diff
@@ -16,7 +16,7 @@ tags:
 title: Add logging to password checker
 ---
 
-In part one of this exercise you created two functions. Now you are going to upgrade that program by adding some logging capabilities.
+In part one of this exercise you created two functions. Now you are going to upgrade that program by adding some logging capabilities using Log4J.
```

Comment:
Some people should be using the python logger. JS people use something else as well.

Please remember that this is nnot only used in the java course. 

You can add extra instructions. Eg: "If you are using java, please use log4J"